sqlgg.opam

```diff
@@ -33,3 +33,7 @@ It starts off with SQL schema and queries, and generates code (or XML, allowing
 further code generation for various purposes). Generated code only defines a mapping
 of output columns and query parameters to the host language, trying to be as unobtrusive
 as possible and leaving the choice of SQL database (and API to access it) to the developer."""
+
+conflicts: [
+  "mariadb" {<= "1.1.6"}
+]
```

Comment:
This won't work with the current deployed versions of `mariadb`

Reply:
with the current opam packaging this is kind of unfortunate - it means people who are using only sqlite traits will still have to deal with this conflict, ideally traits should be each separate opam lib 
i guess ok for this PR :/